content/en/blog/news/jenkins-x-tekton.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Jenkins X  ❤  Tekton"
+date: 2020-03-11
+draft: false
+description: >
+  Like Jenkins X, Tekton is Kubernetes native and has been built from the ground up to leverage Kubernetes. 
+categories: [blog]
+keywords: [Jenkins,Tekton,2020]
+slug: "tekton"
+aliases: []
+author: Ethan Jones
+---
+
+Jenkins X is committing fully to Tekton as its pipeline execution engine. We are convinced that this is the right choice for Jenkins X, as a cloud-native CI/CD platform on Kubernetes, and for our users. 
+
+This means we are formally deprecating - and will be removing - traditional Jenkins static masters support inside Jenkins X. We are excited about the new way forward, which we’ll discuss further below, but it’s important to be clear about what this means for current users. If you are already using Jenkins X with Tekton-based pipelines, then nothing will change for you, and you do not need to change anything. If you’re running a traditional Jenkinsfile on Jenkins X, then you have three choices: 
+
+* Continue using static masters with a version of Jenkins X that supports Jenkins static masters
```

Comment:
Do we plan to provide any security patches for old version of Jenkins X used with static masters? I believe not. Maybe it is worth mentioning this here if this is the case. 

cc @madchap